test/integration/fapi/fapi-policy_signed_delegation.sh

```diff
@@ -80,14 +80,54 @@ BIG_FILE=$TEMP_DIR/big_file.file
 
 # 0. Create Keys
 
-openssl ecparam -name secp256r1 -genkey -noout -out $TEMP_DIR/key_backend_priv.pem
-openssl ec -in $TEMP_DIR/key_backend_priv.pem -pubout -out $TEMP_DIR/key_backend_pub.pem
+# Write private pem key to file
+cat > $TEMP_DIR/key_backend_priv.pem <<EOF
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEICf0OXKKsPkEVR1jsPOKSQQJnJVimamLYwLDZwJDj7etoAoGCCqGSM49
+AwEHoUQDQgAEAw+PKFksCw+ikD76l6BMeXfebcZxGf8QGWT2MOy8tOfpe6m+6MUU
+m2GUijGPkvCTjtJPOJz//XMom+k+7OaWmA==
+-----END EC PRIVATE KEY-----
+EOF
+
+cat > $TEMP_DIR/key_backend_pub.pem <<EOF
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEAw+PKFksCw+ikD76l6BMeXfebcZx
+Gf8QGWT2MOy8tOfpe6m+6MUUm2GUijGPkvCTjtJPOJz//XMom+k+7OaWmA==
+-----END PUBLIC KEY-----
+EOF
+
+cp $TEMP_DIR/key_backend_priv.pem $TEMP_DIR/key_user_priv.pem
+cp $TEMP_DIR/key_backend_priv.pem $TEMP_DIR/key_delegated_priv.pem
+
+cp $TEMP_DIR/key_backend_pub.pem $TEMP_DIR/key_user_pub.pem
+cp $TEMP_DIR/key_backend_pub.pem $TEMP_DIR/key_delegated_pub.pem
+
+# echo "key_backend_priv"
+# cat $TEMP_DIR/key_backend_priv.pem
+
+# echo "key_backend_pub"
+# cat $TEMP_DIR/key_backend_pub.pem
+
+# echo "key_user_priv"
+# cat $TEMP_DIR/key_user_priv.pem
+
+# echo "key_user_pub"
+# cat $TEMP_DIR/key_user_pub.pem
+
+# echo "key_delegated_priv"
+# cat $TEMP_DIR/key_delegated_priv.pem
+
+# echo "key_delegated_pub"
+# cat $TEMP_DIR/key_delegated_pub.pem
+
+# openssl ecparam -name secp256r1 -genkey -noout -out $TEMP_DIR/key_backend_priv.pem
+# openssl ec -in $TEMP_DIR/key_backend_priv.pem -pubout -out $TEMP_DIR/key_backend_pub.pem
 
-openssl ecparam -name secp256r1 -genkey -noout -out $TEMP_DIR/key_user_priv.pem
-openssl ec -in $TEMP_DIR/key_user_priv.pem -pubout -out $TEMP_DIR/key_user_pub.pem
+# openssl ecparam -name secp256r1 -genkey -noout -out $TEMP_DIR/key_user_priv.pem
+# openssl ec -in $TEMP_DIR/key_user_priv.pem -pubout -out $TEMP_DIR/key_user_pub.pem
 
-openssl ecparam -name secp256r1 -genkey -noout -out $TEMP_DIR/key_delegated_priv.pem
-openssl ec -in $TEMP_DIR/key_delegated_priv.pem -pubout -out $TEMP_DIR/key_delegated_pub.pem
+# openssl ecparam -name secp256r1 -genkey -noout -out $TEMP_DIR/key_delegated_priv.pem
+# openssl ec -in $TEMP_DIR/key_delegated_priv.pem -pubout -out $TEMP_DIR/key_delegated_pub.pem
```

Comment:
@cplappert are these debug leftovers?

Reply:
Yes, I was still testing. I removed all debug leftovers now.
As soon as the CI runs successfully, the PR is ready to merge.